Personnummer/Luhn.cs

```diff
@@ -0,0 +1,22 @@
+﻿using System.Linq;
+
+namespace Personnummer
+{
+    public static class Luhn
```

Comment:
Please make sure that the public method/s and class declaration are properly commented.